README.md

```diff
@@ -2,44 +2,54 @@
 
 [![Daily compatibility check against latest polkadot](https://github.com/paritytech/staking-miner-v2/actions/workflows/nightly.yml/badge.svg)](https://github.com/paritytech/staking-miner-v2/actions/workflows/nightly.yml)
 
-WARNING this library is under active development DO NOT USE IN PRODUCTION.
+This is a re-write of the [polkadot staking miner](https://github.com/paritytech/polkadot/tree/master/utils/staking-miner) using [subxt](https://github.com/paritytech/subxt) to avoid hard dependency to each runtime version.
 
-The library is a re-write of [polkadot staking miner](https://github.com/paritytech/polkadot/tree/master/utils/staking-miner) using [subxt](https://github.com/paritytech/subxt)
-to avoid hard dependency to each runtime version. It's using a static metadata (metadata.scale) file to generate rust types, instructions how to update metadata can be found [here](#update-metadata)
+The binary itself embeds [static metadata](./artifacts/metadata.scale) to generate a rust codegen at compile-time that [subxt provides](https://github.com/paritytech/subxt).
 
-The intention is that this library will "only break" once the [pallet-election-provider-multi-phase](https://crates.parity.io/pallet_election_provider_multi_phase/index.html) breaks i.e, not on every runtime upgrade.
+Runtime upgrades are handled by staking-miner-v2 by upgrading storage constants and that will work unless there is a breaking change in any of pallets used by the staking-miner (mainly pallet-election-provider and pallet-system are used).
 
+Because detecting breaking changes when connection to RPC node when using `staking-miner-v2` is hard, this repo performs daily integration tests against `polkadot master` and in most cases updating the metadata [see update metadata](#update-metadata)) and fixing the compile errors are sufficient.
 
+Each release will specify which runtime version it was tested against but
```

Comment:
nit but the break line here is different from the rest of the README.md, maybe we could try to stick with 80chars?